src/components/TBody/VSelect.vue

```diff
@@ -1,14 +1,13 @@
 <template>
   <div v-if="row[header].type === 'select' && row[header].handleSearch">
-    <span :ref="`span-${currentTable}-${colIndex}-${rowIndex}`">{{ row[header].value }}</span>
-    <i class="icon_glass" :class="{ show: row[header].search }" />
-    <button
-      :class="{ active: row[header].search === true }"
-      class="enable_select"
-      @click.stop="enableSelect($event, header, row[header], rowIndex, colIndex)"
-    >
-      <i />
-    </button>
+    <div @click.stop="enableSelect($event, header, row[header], rowIndex, colIndex)">
```

Comment:
For @click event, use `<button type="button">` not a `<div>` element please

Reply:
I made the changes :)
I also updated the select icon to a cleaner approach (avoid using both :after and :before pseudo-elements).